src/web/make-report.rkt

```diff
@@ -15,7 +15,7 @@
       (title "Herbie results")
       (meta ((charset "utf-8")))
       (link ((rel "stylesheet") (type "text/css") (href "report.css")))
-      (script ([src "report-page.js"][type "module"]))
```

Comment:
I remember adding this because of async calls. Cool to see it removed.

Reply:
Yeah `async` in JS is just syntax sugar for promises, and here we use it in just one place. Easy to remove.